nodejs/test/unit/cjsHandler.tap.js

```diff
@@ -27,11 +29,8 @@ tap.test('Layer Handler - CJS Function', (t) => {
     process.env = { ...originalEnv }
     helper.unload()
   })
-  
-  t.test('should wrap handler in transaction', async(t) => {
-    process.env.NEW_RELIC_LAMBDA_HANDLER = 'test/unit/fixtures/cjs/handler.handler'
-    process.env.AWS_LAMBDA_FUNCTION_NAME = 'testFn'
 
+  t.test('should wrap handler in transaction', async(t) => {
```

Comment:
do we need a unit test for lambda in cjs with callback or are we just relying on the integration tests for this?